SU-HSDO/suhumsci
# Fix(STN-841):  Collections paragraph type spacing to match Rows

# READY FOR REVIEW

## Summary
This work updates the collections stylings spacing to match the previous Row stylings.
This should cover more than just the gaps between paragraph types but also match the styling for postcards, testimonials, vertical linked cards, etc.

JIRA Card:
https://sparkbox.atlassian.net/browse/STN-841

This also fixes the image size issues for the additional bug card here: https://sparkbox.atlassian.net/browse/STN-823

## Need Review By (Date)
9/21

## Urgency
asap

## Steps to Test
1. npm run test
2. Create a Flexible page with Rows and different Well width variations and different included elements within.
3. Create a Flexible page with Collections using the same variations and elements and compare the styling.
- Make sure to test different row amounts as spacing is applied to singular element collections and rows then the multiple element rows.
4. Swap between Traditional and Colorful themes and verify the consistency between both is accurate.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
